### PR TITLE
Adding cat command documentation

### PR DESCRIPTION
**Pull request by jattisha**
Commits were working weird so I had to create new repo, lost old commits.

Added cat documentation along with 3 examples.

### CODE REVIEW

**Comment by jattisha** [comment]
#15 was my original issue

**Comment by jattisha** [comment]
Since this was never merged, I'm just curious if I'm doing something wrong so that I can fix it now while I have time.

File: _commands/basics/cat.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+---
+
+CAT
```
Comment (mterwill) [inline _commands/basics/cat.md]: Please make this lowercase

File: _commands/basics/cat.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+}
+//Ctrl+D stops standard input
+
+$ cat main.c
+#include <stdio.h>
+int main() {
+	printf("See how easy this is?");
+	return 0;
+}
+//Ctrl+D stops standard input
```
Comment (mterwill) [inline _commands/basics/cat.md]: I don't know that you want this line (75) here

**Comment by mterwill** [comment]
Sorry, we're a little backed up. Please have a look at the suggestions above. Additionally, please add a section about [heredocs](http://tldp.org/LDP/abs/html/here-docs.html) like @ppannuto has used several times in lecture/homework.

**Comment by jattisha** [comment]
I wanted the // line because if you type it into terminal, that is what comes out. I will make the rest of the changes now, and I will remove 75 if you think it will be more clear.

**Comment by the-alex** [comment]
I like it. If no one else merges, I will.

**Comment by mterwill** [comment]
I like 67 but you don't need to Control + D on 75 because you're just looking at the file you wrote to, yes?

**Comment by the-alex** [comment]
R.I.P. Line 75.

**Comment by mterwill** [comment]
Lolol @the-alex 

and closed #15

**Pull request merged**